RULES_DESCRIPTIONS.md

```diff
@@ -410,6 +411,19 @@ _Description_: Using unit-specific suffix like "Secs", "Mins", ... when naming v
 
 _Configuration_: N/A
 
+## too-many-indirections
```

Comment:
Thinking of it further, looks like this rule discourages chaining. I'm not particularly against this pattern.

I'd suggest to either rename the rule, to be more explicit what it enforces or change the confidence to something below 1.

What do you think?